src/sdk/namespace_fs.js

```diff
@@ -563,11 +564,13 @@ class NamespaceFS {
 
             let count = 1;
             for (let pos = start; pos < end;) {
+                object_sdk.throw_if_aborted();
 
                 // allocate or reuse buffer
                 const remain_size = Math.max(0, end - pos);
```

Comment:
not sure how this line got up here, we can push it a bit lower next to the `const read_size = ...`

Reply:
it's because we wanted to put it at the beginning of every iteration,
and specifically after the open (for the first iteration) and before the get_buffer(),
if i'll move it next to the read_size it won't be checked before the get_buffer()

Reply:
no no no, I meant just the `const read_size = ...` line, not the throw if aborted...